ebuild-writing/functions/src_test/text.xml

```diff
@@ -82,7 +82,6 @@ portage environment. Reasons for such a failure could include:
 </p>
 
 <ul>
-  <li>Needing to use X.</li>
```

Comment:
Feel free to do that. Last I checked, I wasn't on devmanual team or responsible for fixing horribly bad documentation.